wiki/mapping/glossary.md

```diff
@@ -16,232 +16,242 @@ let us know in #mapping-discussion!
+| **Lighting Track** | The area in most map editors where [Lighting](#l) events are placed. See also: [Track](#t). |
+| **Lightshow Mode** | A map characteristic where there are no [Blocks](#b), only [Lighting](#l). This mode is not part of the base game and requires the [SongCore](#s) mod. |
+| **Loloppe Notes** | Two same-direction [Blocks](#b) placed side-by-side such that hitting both blocks requires abusing the block [Hitbox](#h). Named after the mapper who made them famous. <details><summary>**Example Image**</summary>![Picture of loloppe notes](~@images/mapping/loloppe.png)</details> |
 
 ## M
 | Term&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; | Definition |
 | - | :- |
-| **Map Blindness** | Mapper blindness arises when a mapper has spent so much time on a map. You know the patterns, you know your intentions. You've self tested it so many times. Eventually, muscle memory and subconscious kicks in, and you can play the map with your eyes closed, regardless of flaws. |
-| **Mapping Extensions** | A mod developed by Kyle1314 that allows mappers access to a number of unique utilities for advanced mapping. See also [Precision Placement](#p), [Precision Rotation](#p), [JSON Walls](#jk) |
-| **Mediocre Map Assistant 2** | A new fork of Mediocre Mapper by Assistant. The primary map editor used by the mapping community. |
-| **Mediocre Mapper** | An outdated editor, developed by squeaksies as a fork of the original EditSaber editor by Ikiewa. No longer in public development or supported by its author it has been forked to created **Mediocre Map Assistant 2**. |
-| **Mine** | A *block* that looks like a spiky circle and that damages the player if their saber touches it. It is safe for a mine to pass through the player’s body. Also known as [Bomb](#b) |
+| **Map Blindness** | A condition that may occur when a mapper has spent so much time on a map. You know the patterns, you know your intentions, and you've self-tested it so many times. Eventually, muscle memory and your subconscious kick in, and you can play the map with your eyes closed, regardless of flaws. See also: [Playtest](#t). |
+| **Mapping Extensions** | A mod developed by Kyle1314 that allows mappers to access a number of unique utilities for advanced mapping. See also: [JSON Walls](#jk), [Precision Placement](#p), [Precision Rotation](#p). |
+| **Mediocre Map Assistant 2** | A new fork of [Mediocre Mapper](#m) by Assistant. This is the primary map [Editor](#e) used by the mapping community. |
+| **Mediocre Mapper** | An outdated [Editor](#e), developed by squeaksies as a fork of the original EditSaber editor by Ikiewa. No longer in public development or supported by its author, it has been forked to created [Mediocre Map Assistant 2](#m). |
+| **Mine** | A [Block](#b) that looks like a spiky circle that damages the player if their saber touches it. It is safe for a bomb to pass through the player’s body. Also known as [Bomb](#b). |
```

Comment:
Suggest using mine in the definition of mine instead of bomb.
```suggestion
| **Mine** | A [Block](#b) that looks like a spiky circle that damages the player if their saber touches it. It is safe for a mine to pass through the player’s body. Also known as [Bomb](#b). |
```